### PR TITLE
Feature/fix reports bugs

### PR DESCRIPTION
**Pull request by LisaQ04**
Fixed bugs related to reports and graphs.

### CODE REVIEW

File: waterlog/src/Components/Usage.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ class Usage extends Component {
       return <div>
         <p><b>Cost</b></p>
         <DailyCostsReports props={this.props.dailyCost} />
+        <br/>
```
Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/Components/Usage.js]: Rather use styling to space the components.

Comment (LisaQ04) [reply waterlog/src/Components/Usage.js]: I added style.

File: waterlog/src/App.scss [hunk 2]
```diff
@@ -13,17 +13,24 @@
 #cost-buttons {
 	padding-top: 18px; 
 }
-
 .btn-graph-nav{
 	height:26px; 
 	width: 84px;
 }
- 
+#divide{
```
Comment (FredJEhlers) [inline waterlog/src/App.scss:22]: empty lines between declarations (Do for entire file)

File: waterlog/src/Components/MonthlyUsage.js [hunk 3]
```diff
@@ -29,13 +29,19 @@ const MonthlyUsageComponent = (props) => {
             fontStyle: 'bold',
             fontColor: '#FF0000'
```
Comment (FredJEhlers) [inline waterlog/src/Components/MonthlyUsage.js]: Stick to hex or rgba

File: waterlog/src/Components/SeasonalUsage.js [hunk 4]
```diff
@@ -31,13 +31,19 @@ const SeasonalUsageComponent = (props) => {
             fontStyle: 'bold',
             fontColor: '#FF0000'
```
Comment (FredJEhlers) [inline waterlog/src/Components/SeasonalUsage.js]: Stick to hex or rgba

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/App.scss (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/App.scss:22]: Left lines for entire file

File: waterlog/src/Components/MonthlyUsage.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/Components/MonthlyUsage.js]: now using rgba.

File: waterlog/src/Components/SeasonalUsage.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/Components/SeasonalUsage.js]: now using rgba.

**Review by Tebogo-Seshibe-Retro — APPROVED** [review]

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**